src/js/components/ParseErrors.jsx

```diff
@@ -73,8 +75,9 @@ const ParseErrors = (props) => {
 
 ParseErrors.propTypes = {
   total: PropTypes.number,
-  transmittalSheetErrors: PropTypes.array,
-  larErrors: PropTypes.array
+  transmittalSheetErrors: PropTypes.array.isRequired,
+  larErrors: PropTypes.array.isRequired,
+  isFetching: PropTypes.bool.isRequired
```

Comment:
Should we do something with `isFetching`? Like [this](https://github.com/cfpb/hmda-platform-ui/blob/52f6da46854d69acc3bff415e0658a8b2350e22a/src/js/components/Institutions.jsx#L157).